posts/leveraged-learning-with-chat-gpt.md

```diff
@@ -0,0 +1,101 @@
+
+![Blooms Taxonomy](../images/2023/05/blooms-taxonomy.png)
+
+The idea here is for example that just “remembering” or “understanding” and being to explain a concept is different from being able to create or analyse a concept for a greater depth of understanding, but at the same time the lower levels are still prerequisites in order to reach the higher levels of mastery.
+
+In the context of programming, even though it is often tempting and probably very popular to be copy pasting code directly from stackoverflow, we really should have a decent understanding of the concept at least from a high level about what we are actually working with - otherwise we can go down a route with unforeseen errors that could have otherwise been avoided.
+
+Now the difficult part here is that we must manually find relevant resources in order to gain the initial understanding and remember phase - where Google is the goto. The first question I always have is based around **why. **Why is this piece of technology important, how does it fit with my pre-existing knowledge or beliefs and why is it important? Or maybe its not relevant at the time but I still need to know more about it. But with that we are left with a few problems - finding answers that are a combination of **consistent, reliable **and **relatable.** What I mean by that is, maybe we find the perfect Stackoverflow question that answers the exact question we had in mind - but as we all know Stackoverflow answers vary considerably in quality and we can’t always expect a high level answers that always makes sense to every single person, we need to go through multiple sources and compile a bunch of information in order to really get a good understanding here. Lastly, relatability is important because it allows us to fulfil parts of the “analyse” step in Blooms Taxonomy. The idea here is to leverage our existing knowledge in order to learn concepts more easily - one example is how we learn the “queue” data structure: because it follows the same concept as how a queue works like any line at a fast food restaurant, the concept of LIFO (last in, first out) becomes a lot easier to grasp than it would have been if we had to learn from complete scratch.
+
+<p id="gdcalert2" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image2.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert3">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
+
+![LIFO](../images/2023/05/lifo.png)
```

Comment:
What's the license on this image? Is this something you've created, or pulled from somewhere else. We effectively can only use royalty free images for this.